start-page/release/index.html

```diff
@@ -2,45 +2,82 @@
-        </section>
-        <section>
-          <h2><i class="fas fa-hands-helping fa-lg" style="color:#ed00b5"></i> {{ _('<a href="%(contribute)s">Get Involved</a>')|format(contribute='https://www.thunderbird.net/get-involved/') }}</h2>
-          <p>{{ _('Thunderbird is an open source project, which means anyone can contribute ideas, designs, code, and time helping fellow users. Become part of a team that supports Thunderbird users, and makes Thunderbird awesome. <a class="more" href="%(contribute)s">Join us</a>')|format(contribute='https://www.thunderbird.net/get-involved/') }}</p>
-        </section>
-      </div>
-      <div>
-        <section>
-          <h2><i class="far fa-life-ring fa-lg" style="color:#0060df"></i> {{ _('<a href="%(support)s">Get Support</a>')|format(support='https://support.mozilla.org/products/thunderbird') }}</h2>
-          <p>{{ _('Need help with something in Thunderbird? We have a <a href="%(support)s">support page</a> where you can ask questions and read knowledge base articles to learn how to use Thunderbird.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
-        </section>
-        <section>
-          <h2><i class="fas fa-bug fa-lg" style="color:#058b00"></i> {{ _('<a href="%(bug)s">Report a Bug</a>')|format(bug='https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird') }}</h2>
-          <p>{{ _('Think you’ve found a problem with Thunderbird? <a href="%(bug)s">Create a bug report</a> so that we can make sure it gets solved. Helping us identify and fix bugs makes for a better Thunderbird for all.')|format(bug='https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird') }}</p>
-        </section>
+
+      <div class="main-content">
+        <div class="content-sections">
+          <section id="get-involved">
+            <h2 class="section-heading get-involved">
+              {{ svg('handshake') }}
+              {{ _('Help') }}
+            </h2>
+            <p>{{ _('Curious about helping? You don’t need to be an expert or know how to code to help other users or offer suggestions about improving the experience! Join us in this open source project where anyone can participate in the Thunderbird community.') }}</p>
+            <a href="https://www.thunderbird.net/en-US/get-involved/" class="get-involved">
+              {{ _('Get Involved') }}
+              {{ svg('external-link') }}
+            </a>
+          </section>
+
+          <section id="crash">
+            <h2 class="section-heading crash">
+              {{ svg('info') }}
+              {{ _('Crashed?') }}
+            </h2>
+            <p>{{ _('If Crash Reporter appears when Thunderbird crashes, please provide as many details as possible to help developers') }}</p>
```

Comment:
Nit: add a period at the end of the sentence.

```suggestion
            <p>{{ _('If Crash Reporter appears when Thunderbird crashes, please provide as many details as possible to help developers.') }}</p>
```